tests/rtl/hardware_test.py

```diff
@@ -385,6 +388,73 @@ def test_hardware_axpy_double_pump_vec4():
     return test_hardware_axpy_double_pump(veclen=4)
 
 
+@rtl_test()
+def test_hardware_vadd_temporal_vectorization():
+    # TODO !!!!! THIS TEST STALLS WITH VITIS 2021.2 !!!!!
```

Comment:
oh dear lord

Reply:
( ͡° ͜ʖ ͡°)

Reply:
Do you have an idea why?
Maybe you can a short comment on this

Reply:
It seems like it's the last transaction somewhere inside that messes up, as it receives all of it's input, but the output is missing a single transaction. It looked like it was the floating point adder was the culprit in hardware emulation. I would have to double check though, in order to be certain to this claim.